src/ORM/DataQuery.php

```diff
@@ -752,12 +752,8 @@ public function reverseSort()
 
     /**
      * Set the limit of this query.
-     *
-     * @param int $limit
-     * @param int $offset
-     * @return $this
      */
-    public function limit($limit, $offset = 0)
+    public function limit(?int $limit, int $offset = 0): static
```

Comment:
Not necessary to meet the ACs, but seems logical to change this while we are at it.